client/src/components/teams/MembersToBeInvited.js

```diff
@@ -0,0 +1,67 @@
+import {
+  Typography,
+  Grid,
+  Button,
+  makeStyles,
+  InputLabel,
+} from "@material-ui/core";
+import { Brightness1 } from "@material-ui/icons";
+import React from "react";
+
+const MembersToBeInvited = ({ teamMembers, setTeamMembers }) => {
+  const classes = useStyles();
+  const teamMemberNotEmpty = teamMembers.length !== 0;
+  const removeEmailFromInvitedList = (memberToRemove) => () => {
+    // Caveat: each email must be unique in the list for this to work
+    const filteredList = teamMembers.filter(
+      (member) => member.email !== memberToRemove.email
+    );
+    setTeamMembers(filteredList);
+  };
+  return (
+    <Grid container direction="column" className={classes.root}>
+      <Grid item>
+        <InputLabel className={classes.label}>
+          An invitation email will be sent to the following members:
+        </InputLabel>
+      </Grid>
+      <Grid item container direction="column" wrap="nowrap">
+        {teamMemberNotEmpty &&
```

Comment:
If the teamMembers array is empty, the map doesn't display anything so there's no need for the check.
Also since `teamMemberNotEmpty` is used only once, we could've just done `teamMembers.length > 0 &&`

Reply:
If the array is empty I think we would get an error when doing `[].map(member=>{member.email})` that's why we included the length check, but inlining it makes sense!